src/python/pants/backend/python/dependency_inference/module_mapper_test.py

```diff
@@ -59,6 +64,18 @@ def test_default_module_mapping_uses_tuples() -> None:
         assert len(modules) > 0
 
 
+def test_default_module_pattern_mapping_keys_and_value_types() -> None:
+    for pattern, replacements in DEFAULT_MODULE_PATTERN_MAPPING.items():
+        assert isinstance(pattern, re.Pattern)
+        assert isinstance(replacements, Iterable)
+        for replacement in replacements:
+            assert (
+                isinstance(replacement, FunctionType)
+                or isinstance(replacement, str)
+                or callable(replacement)
```

Comment:
**comment:** this is mostly to catch partial functions
Couldn't find a concise way to capture partials(which are being used), so `callable()` should work.